### PR TITLE
french translation for LXImage

### PR DESCRIPTION
**Pull request by Super-Baleine**
XImage french translation is done. :whale:

### CODE REVIEW

File: lximage-qt/lximage-qt_fr.ts [hunk 1]
```diff
@@ -0,0 +1,408 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1" language="ca">
```
Comment (palinek) [inline lximage-qt/lximage-qt_fr.ts]: Language code in file name doesn't match the one provided here.

**Comment by Super-Baleine** [comment]
@palinek , done.

**Comment by Super-Baleine** [comment]
Hey, someone can merge please  ? :whale:

**Comment by palinek** [comment]
1. is there a reason for not translating all the strings?
2. all provided translations have the "unfinished"  type, is this intentional?
3. please, update your newly added .ts file as it was done in @493b9fedfc2dd153ac9b6f3d7c3f2b86616ee1cf
4. after finished, please, squash the commits and give a suitable log message for it

**Comment by Super-Baleine** [comment]
Oh yea ! Sry, i forgot it !

**Comment by Super-Baleine** [comment]
Also, the hotkeys are the same in all languages.. i think it's useless to write it in the translation

**Comment by Super-Baleine** [comment]
Normally, all is good !

**Comment by palinek** [comment]
There are 6 commits in this PR already, so 

> after finished, please, squash the commits and give a suitable log message for it

is still needed.

**Pull request closed without merging**